R/variable.R

```diff
@@ -205,6 +205,17 @@ unbind <- function (x) {
     invisible(out)
 }
 
+#' @rdname webApp
+#' @export
+#' @importFrom utils browseURL
+setMethod("webApp", "CrunchVariable", function (x) browseURL(APIToWebURL(x)))
+
+setMethod("APIToWebURL", "CrunchVariable", function (x) {
+    url <- gsub("(app.crunch.io\\/api\\/datasets)", "app.crunch.io/dataset", self(x))
+    url <- gsub("\\/variables\\/", "\\/browse?variableId=", url)
```

Comment:
If you change this to `gsub("^(.*)/variables/(.*?)/.*$", "\\1/browse?variableId=\\2", url)`, you don't need the next line, and you get some insurance in case the variable entity grows query parameters or something. 

Alternatively, you could also construct the URL rather than use regular expressions. The variable link here is the dataset link + `browse?variableId=` + `id(x)`. 

Side note: you don't need to `\\` escape `/` in your regexp, and you don't need to escape anything in the replacement text since it isn't a regexp.